playbooks/provisioning/openstack/provision-openstack.yml

```diff
@@ -8,6 +8,10 @@
     - include: pre_tasks.yml
   roles:
     - role: openstack-stack
+    - role: static_inventory
+      when: openstack_inventory|default('static') == 'static'
+      inventory_path: "{{ openstack_inventory_path|default('../../../../inventory') }}"
```

Comment:
This will only work with the default inventory at `~/inventory` and needs to be modified every time that location is changed. Your deployers will likely maintain multiple inventories and will be passing them to ansible-playbook via `-i path/to/inventory`.

If you change the line to this:

    inventory_path: "{{ openstack_inventory_path|default(inventory_dir) }}"

It will work both with the current `~/inventory` situation from README and with any value passed to `-i`.